azerothcore/azerothcore-wotlk
# Core/ICC: Lady Deathwhisper

##### CHANGES PROPOSED:

-  Added missing overrides and changed NULL for nullptr.
-  Fix timers (this needs atention, it might be wrong)
-  No more Dark Martyrdom being casted in phase 2.
-  Added temporal immunities to certain spell mechanics to avoid interrupt Dark Martyrdom cast.
-  Added the correct auras to reanimated cultists (fanatic's/adherent's determination).
-  Avoid spam of Dark Martyrdom.
-  Lady Deathwhisper should be immune to taunt on heroic.


###### ISSUES ADDRESSED:


Closes #1538 
Closes #761 
Maybe updates #1532 


##### TESTS PERFORMED:


Compiled and tested. Need more tests though.


##### HOW TO TEST THE CHANGES:


Go to icc, kill Lord Marrowgar and go to Lady Deathwhisper. Play with mobs during first phase, make sure you can't interrupt Dark Martyrdom's cast, etc... Detailed above.

##### KNOWN ISSUES AND TODO LIST:



##### Target branch(es):

Master
